mod/05_email/format/email_address_format.rb

```diff
@@ -0,0 +1,15 @@
+# -*- encoding : utf-8 -*-
+
+class Card::EmailAddressFormat < Card::TextFormat  
+  def link_to  text, href, opts={}
+    if text and href != text
+      "#{text} <#{href}>"
+    else
+      href
+    end
+  end
+
+  def chunk_list
+    :references
+  end
+end
```

Comment:
I've just introduced that format  but I'm going to remove it (unless there is somebody who wants to defend it).
The only difference compared to EmailText is the link_to method and this doesn't make a lot of sense, at least not how links currently work. It was supposed to render [[my@mail.com | name]] as 'name <my@mail.com>'  but Wagn interprets  my@mail.com as a card name.  One could make [[mailto:my@mail.com | name]] work. link_to has to remove the 'mailto:' from the href variable for that. But I doubt that this is useful.  I think it's more convenient to write directly 'name <my@mail.com>'